src/travis2docker/travis2docker.py

```diff
@@ -61,6 +61,8 @@ def mkdir_p(path):
     def __init__(self, yml_buffer, image=None, work_path=None, dockerfile=None,
                  templates_path=None, os_kwargs=None, copy_paths=None,
                  ):
+        self._python_versions = ['2.7']
+        self._supported_versions = ('2.7' '3.2' '3.3' '3.4' '3.5' '3.6')
```

Comment:
We shouldn't manage supported versions from t2d since that we can use many images.
We should get a error if the image dont have the environment defined in the source command.

Reply:
@moylop260 
Done, now the travis2docker don't not check the versions of python